shared/src/commonMain/kotlin/org/hyperskill/app/study_plan/widget/presentation/StudyPlanWidgetFeature.kt

```diff
@@ -68,6 +69,17 @@ object StudyPlanWidgetFeature {
         object ReloadContentInBackground : Message
 
         object PullToRefresh : Message
+
+        /**
+         * Show stage implement unsupported modal
+         */
+        object StageImplementUnsupportedModalGoToHomeClicked : Message
```

Comment:
I suppose should be moved below, under `StageImplementUnsupportedModalHiddenEventMessage`

Reply:
and comment "Show stage implement unsupported modal" could be deleted

Reply:
This message is used also for navigation. Depending on existing approach, we don't move such messages to analytic block